packages/wallets/ledger/package.json

```diff
@@ -27,6 +27,7 @@
         "postbuild": "echo '{\"type\":\"commonjs\"}' | npx json > lib/cjs/package.json && echo '{\"type\":\"module\"} ' | npx json > lib/esm/package.json"
     },
     "dependencies": {
+        "buffer": "npm:buffer@^6.0.3",
```

Comment:
Used `buffer` instead of the olde `buffer-browserify` which didn't have the full API surface needed by Ledger.